lib/components/Tabs/TabList.tsx

```diff
@@ -15,7 +23,9 @@ export const TabList: FunctionComponent = ({ children }) => {
 			borderWidthBottom="1"
 			borderColour="gray"
 			role="tablist"
-			className={styles.tabsList}>
+			className={clsx(styles.tabsList.root, {
+				[styles.tabsList.stretch]: stretch,
+			})}>
 			{Children.map(children, (child, index) => (
 				<IndexContext.Provider value={index}>
```

Comment:
Will need to add flex auto to children when stretching too.

Reply:
Ah, so the items not only position them Left and Right. But also "stretch". Great pick up mate :100: